From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Igor Pecovnik <igor.pecovnik@gmail.com>
Date: Sun, 26 Aug 2018 14:14:14 +0200
Subject: [ARCHEOLOGY] Creating new family rockchip64. 3288 and 3399 was merged
 in, except Nanopi which stays in RK3399 family with sources attached to FA
 repository until we manage to join that in. Enabled Nanopc T4, Nanopi M4,
 Renegade. Tested on Rock64 and Nanopc T4.

> X-Git-Archeology: > recovered message: > Some clenaup, removed dragonboard820c, adjusted board descriptions, added another OPP for 3288 family,
> X-Git-Archeology: - Revision 41b589372f2c1a8450033db78f3d8ed9db1b2196: https://github.com/armbian/build/commit/41b589372f2c1a8450033db78f3d8ed9db1b2196
> X-Git-Archeology:   Date: Sun, 26 Aug 2018 14:14:14 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igor.pecovnik@gmail.com>
> X-Git-Archeology:   Subject: Creating new family rockchip64. 3288 and 3399 was merged in, except Nanopi which stays in RK3399 family with sources attached to FA repository until we manage to join that in. Enabled Nanopc T4, Nanopi M4, Renegade. Tested on Rock64 and Nanopc T4.
> X-Git-Archeology:
> X-Git-Archeology: - Revision 08614f398205625151ea5fa1d995734110b45fe1: https://github.com/armbian/build/commit/08614f398205625151ea5fa1d995734110b45fe1
> X-Git-Archeology:   Date: Fri, 11 Jan 2019 11:43:35 +0100
> X-Git-Archeology:   From: JMCC <JMCC@localhost>
> X-Git-Archeology:   Subject: Fix 1512MHz OPP for Renegade
> X-Git-Archeology:
> X-Git-Archeology: - Revision 150ac0c2afa147d9e3b036c8ecd8238fe5648cf3: https://github.com/armbian/build/commit/150ac0c2afa147d9e3b036c8ecd8238fe5648cf3
> X-Git-Archeology:   Date: Tue, 19 Nov 2019 23:25:39 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Remove K<4, change branches, new features (#1586)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 0cdffb29b07305209efb12cf3b5ac6032d3a1153: https://github.com/armbian/build/commit/0cdffb29b07305209efb12cf3b5ac6032d3a1153
> X-Git-Archeology:   Date: Wed, 24 Mar 2021 19:01:53 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Renaming DEV branch to EDGE (#2704)
> X-Git-Archeology:
---
 arch/arm64/boot/dts/rockchip/rk3328-roc-cc.dts | 7 +++----
 arch/arm64/boot/dts/rockchip/rk3328.dtsi       | 7 +++++++
 2 files changed, 10 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3328-roc-cc.dts b/arch/arm64/boot/dts/rockchip/rk3328-roc-cc.dts
index dee5a395fb55..bfacc080d5de 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328-roc-cc.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3328-roc-cc.dts
@@ -360,11 +360,10 @@
 		clock-latency-ns = <40000>;
 	};
 	opp-1512000000 {
-		status = "disabled";
 		opp-hz = /bits/ 64 <1512000000>;
-		opp-microvolt = <1350000>;
-		opp-microvolt-L0 = <1350000>;
-		opp-microvolt-L1 = <1325000>;
+		opp-microvolt = <1450000>;
+		opp-microvolt-L0 = <1450000>;
+		opp-microvolt-L1 = <1425000>;
 		clock-latency-ns = <40000>;
 	};
 };
diff --git a/arch/arm64/boot/dts/rockchip/rk3328.dtsi b/arch/arm64/boot/dts/rockchip/rk3328.dtsi
index 3ab51435101e..ac71ce493833 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3328.dtsi
@@ -172,6 +172,13 @@
 			opp-microvolt-L1 = <1325000 1325000 1325000>;
 			clock-latency-ns = <40000>;
 		};
+		opp-1512000000 {
+			opp-hz = /bits/ 64 <1512000000>;
+			opp-microvolt = <1450000 1450000 1450000>;
+			opp-microvolt-L0 = <1450000 1450000 1450000>;
+			opp-microvolt-L1 = <1425000 1425000 1425000>;
+			clock-latency-ns = <40000>;
+		};
 	};
 
 	arm-pmu {
-- 
Armbian

